packages/web-components/src/components/va-checkbox/va-checkbox.scss

```diff
@@ -95,6 +95,7 @@
   background-image: url('../../assets/check.svg');
   background-position: 50%;
   background-repeat: no-repeat;
+  background-size: cover;
```

Comment:
This is for `:host(:not([uswds])) input:checked+label::before` which is the v1 version. I'm not sure why it is needed if this PR is for v3.

Reply:
You raise a good point. The selector :`host(:not([uswds])) input:checked+label::before` was used in the v1 version and seems to have carried over to v3. However, I agree that it may not be necessary for v3 and I have also noticed that some CSS properties are being applied to V3 that shouldn't be. FYI, [V3 checkbox](https://design.va.gov/storybook/?path=/docs/uswds-va-checkbox--default)  in production have small check icon. Let me review it again.

Reply:
That is interesting. I'm not sure when the check got so small like that. 🤔 

![Screenshot 2023-04-20 at 4 51 30 PM](https://user-images.githubusercontent.com/872479/233495345-b122f763-c231-46dc-bb46-dcf47c1ab9d0.png)

It would be great if you were able to figure out how to fix that in this PR. But it is also reasonable if you just create a separate ticket for it. It does not seem to be an issue when it's loaded in the v3 `va-privacy-agreement`.

Reply:
I was able to fix it by adding `reflect` to `uswds` under checkbox component.